services/user/user.go

```diff
@@ -13,7 +14,10 @@ import (
 
 func CreateUser(ctx context.Context, user user_model.User) (*user_model.User, error) {
 	db := gorm_agent.GetDefaultDB()
-
+	_, err := judge_model.CreateJudgeRankCache(db, judge_model.NewJudgeRankCache(user.Account))
```

Comment:
If RankCache is necessary for every user, using relationship in GORM will be better, so that we won't need to create RankCache explicitly.

Reply:
I'm also welcome the design of lazy creation for RankCache, this depends on whether we think RankCache is needed for user who don't submit any judges.

Reply:
Now it's lazy creation I think.